### DISCUSSION TITLE
Where is code editor in playground

### DISCUSSION THREAD

**Discussion by jcubic** [motion-canvas/motion-canvas#1131] [Q&A]
I wanted to test, but I don't see a way to edit. I've opened Quick Start and there is fixed animation without the source code.

![Przechwycenie obrazu ekranu_2024-11-11_12-30-28](https://github.com/user-attachments/assets/f5dc402e-dc14-41f3-93ca-6ece95420229)

How to access the source code? Do I need to install it to be able to edit?

**Comment by milotheirself** [comment]
Hello @jcubic! When you initialize a project, it'll create a directory that contains all the source files with the name of your project. The source files related to the animation itself can be found under `src/scenes`, `src/scenes/example.tsx` in case of the example scene in a new project, as mentioned under [programming an animation](https://motioncanvas.io/docs/quickstart/#programming-an-animation) in the quickstart documentation page.

**Reply by jcubic** [reply]
So the playground don't have any way to play with the code, isn't it?

**Reply by milotheirself** [reply]
You edit source files in your own code editor locally. If you take a look at [time events](https://motioncanvas.io/docs/time-events) or the more complex scenes in the [examples repo](https://github.com/motion-canvas/examples), you may see the function of the editor (which I think you mean by playground) more clearly. You don't directly create animations in the editor, that happens in the source files, you only time them to other assets like video or audio files and look at it as a preview of your output.

**Reply by jcubic** [reply]
You didn't answer there is no playground on the website, there is one in documentation, but people first visit a demo, that you can't edit. You don't even see the code, which is kind of useless for a demo.
